src/components/App/App.js

```diff
@@ -159,6 +266,24 @@ function App() {
   }, []);
 
   React.useEffect(() => {
+    mainApi
+      .getSavedArticles(localStorage.getItem('token'))
+      .then((articles) => {
+        setSavedCards(articles);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+
+    mainApi
+      .checkToken(localStorage.getItem('token'))
+      .then((user) => {
+        setCurrentUser(user);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+
     getScreenWidth();
     window.addEventListener('resize', getScreenWidth);
 
```

Comment:
` return window.addEventListener('resize', getScreenWidth);`

the same mistake with the **clean up**

you need to use `removeEventListener` here and pass a **function**



```js

    return () => window.removeEventListener('resize', getScreenWidth);


```